### DISCUSSION TITLE
Intro to Andrew's Blog

### DISCUSSION THREAD

**Discussion by giscus** [Andrew-Chen-Wang/blog#2] [General]
# Intro to Andrew's Blog

What I want this blog to be

https://blog.andrewcwang.com/blog/2024-06-29-intro-to-andrew/

**Comment by Andrew-Chen-Wang** [comment]
I'm reconsidering how I view software too; yes, it's a game changer. But I used to not think it was innovative. I have always believed innovation trumps inventiveness from a capitalist perspective (both are equally important societally speaking). But software, like innovating any form of mechanical or electrical engineering, is a form of improvement.

I see inventions and innovations as a staircase. Each step is an inventor laying the ground work, and the feet is the innovation built around it, accelerating what's imaginable until the level of improvement plateaus and new inventions are needed.

**Reply by Andrew-Chen-Wang** [reply]
To clarify, software can only make things more efficient and automated -- **by definition**. People saying "AI will take over the world" don't understand the nuance of B2B software which is inherently a human inefficiency problem. A lot of B2B SaaS is fixing inefficient human-to-human processes.

Innovation is in software is about finding the most efficient route for stuff. Whether that be from the B2B SaaS bundling revolution demonstrated by Rippling or integration startups (app approach like Lumos, standardization approach like Merge, employee approach like Rippling -- all of these take advantage of VC money because of how labor intensive integrations are) or any other random SaaS, all of them have the end goal of efficiency.

For other industries, it's not making something more efficient. Sure, innovating the car could be about fuel *efficiency*, but cars' innovations can target outside of efficiency such as car seatbelts and airbags which is to improve safety.

**Reply by Andrew-Chen-Wang** [reply]
I think I should also clarify innovation vs invention.

Both are equally societally important. I've always viewed invention as an innovation in it of itself and only becomes an "invention" when it has lasting impact. The car airbag is not originally credited to being for cars, but its invention has led to its widespread usage. In the context of material science, the necessity for stealth paint can be considered an invention because the materials to form the compounds are radical -- yet they too are innovations on what we have previously known.

Regardless of this philosophical question around the difference between invention and innovation, simply speaking: invention could arguably stated as something new and innovation as optimizing it.

Inventions are everywhere and are vital for a healthy advancing society. But for a technologically advancing society, innovation is the economic powerhouse that dictates what civilization will look like in the future. The car is optimized so well, our globe spans millions of miles of highways because of the auto industry's legacy. Optimizing great inventions -- innovating -- solves human problems, which is plenty and the entire purpose of capitalist companies, but always improves the livelihoods of people.

**Comment by Andrew-Chen-Wang** [comment]
I think the "decrease time cooking food" could honestly be replaced with "replace food costs" in general. The average person shouldn't listen to "your time is invaluable"; that's true, and I follow by the principles, but if you work a 9-5, cook your food because your paycheck isn't increasing (statistically true as it's not keeping up with inflation).

I do think the core issue is just food cost. If we were to replace food cost in terms of preparation, restaurants are expensive because of labor and rent -- which is where most of restaurants' margins disappear.

When one visits NYC, some are shocked by prices of food carts and how delicious they can be (and a lot of the times, better than restaurants). I'm still trying to open a halal cart in UMich's campus to provide cheap food to students with decent margins to pay immigrant employees better.

But that's just the thing: UMich doesn't allow food carts within campus for traffic reasons, pedestrian and primarily vehicle congestion. We've given up so much of our space to cars that we smush a lot of activities to indoor rooms when the outdoors with fresh air should be viable. Take a look at Barcelona's [Superilles](https://www.youtube.com/watch?v=ZORzsubQA_M). Foot traffic and economic activity increased with a concentration of social activity because of the freed up space by taking out cars.

If we have more street vendors (in addition to avoiding taxes through cash but not health inspections of as many vendors in NYC still have health grades), we can increase competition and lower price.

Although cities could make construction friendly zoning to prevent skyhigh property valuation and rent, rent is still such a large expense which also takes up a lot of space such that competition is still too limited.

**Comment by Andrew-Chen-Wang** [comment]
So much to talk about in a one-pager re American city planning for cultural and economic growth, but from my POV in NYC, it's actually much faster to live in a suburb via car. I live 7 minutes away from the subway and 10 minutes from a high quality grocery store, but the walking is pretty tiring. Sometimes, just to meet someone, it takes 30-40 minutes once you take into account the subway delays and waiting. No one takes busses because it stops too frequently (for the elderly to go 2-3 stops to decrease their walking time). I want to visit China and Japan to see if it's faster to get around there than NYC and a suburb.
